src/platform/tls_schannel.c

```diff
@@ -1376,6 +1380,14 @@ CxPlatTlsSecConfigCreate(
 
     if (CredConfig->Flags & QUIC_CREDENTIAL_FLAG_REQUIRE_CLIENT_AUTHENTICATION) {
         Status = CxPlatTlsSetClientCertPolicy(AchContext->SecConfig);
+        if (QUIC_FAILED(Status)) {
+            QuicTraceEvent(
+                LibraryErrorStatus,
+                "[ lib] ERROR, %u, %s.",
+                Status,
+                "CxPlatTlsSetClientCertPolicy");
```

Comment:
The error log is unnecessary, because `CxPlatTlsSetClientCertPolicy` already logs the first point of failure.